engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -435,6 +437,15 @@ public static String isValidImportString(Logger log, String importString) {
         }
     }
 
+    private static boolean visibleToClassGraph(String packageImport) {
+        try (ScanResult scanResult = new ClassGraph().enableClassInfo().acceptPackages(packageImport).scan()) {
+            final Optional<ClassInfo> firstClassFound = scanResult.getAllClasses().stream().findFirst();
+            // force load the class so that the jvm is aware of the package
+            firstClassFound.ifPresent(ClassInfo::loadClass);
```

Comment:
Is this going to invoke static initializers for the class? Could this have undesirable side-effects?
(I think the answer is "no", since you have not called `io.github.classgraph.ClassGraph#initializeLoadedClasses` on the `ClassGraph` instance, but it might be worth testing/verifying. Also, `ClassGraph` falls back to `Class.forName()`, which defaults to `true` for initializing classes, which is kind of broken if you ask me, and maybe we should offer them (`ClassGraph`) a PR to fix it.)
That said, our `classExists` is not taking any such precautions. Maybe we need to make these things more consistent

Reply:
If you dig in deeper, it actually doesn't fall back to forName (I think) - by default, it calls w/ `initializeLoadedClasses=false` and the explicit forName call.

Reply:
```java
    Class<?> loadClass(final boolean ignoreExceptions) {
        if (classRef == null) {
            final String className = getClassInfoNameOrClassName();
            if (scanResult != null) {
                classRef = scanResult.loadClass(className, ignoreExceptions);
            } else {
                // Fallback, if scanResult is not set
                try {
                    classRef = Class.forName(className);
                } catch (final Throwable t) {
                    if (!ignoreExceptions) {
                        throw new IllegalArgumentException("Could not load class " + className, t);
                    }
                }
            }
        }
        return classRef;
    }
```

I think `scanResult` will always be non-null.

```java
    public Class<?> loadClass(final String className, final boolean returnNullIfClassNotFound)
            throws IllegalArgumentException {
        if (closed.get()) {
            throw new IllegalArgumentException("Cannot use a ScanResult after it has been closed");
        }
        if (className == null || className.isEmpty()) {
            throw new NullPointerException("className cannot be null or empty");
        }
        try {
            return Class.forName(className, scanSpec.initializeLoadedClasses, classGraphClassLoader);
        } catch (final ClassNotFoundException | LinkageError e) {
            if (returnNullIfClassNotFound) {
                return null;
            } else {
                throw new IllegalArgumentException("Could not load class " + className + " : " + e, e);
            }
        }
    }
```

Reply:
I'm suggesting we change our code to do:
```
Class.forName(className, false, GroovyDeephavenSession.class.getClassLoader());
```